source/tests/consistent/test_stat_file.py

```
@@ -0,0 +1,224 @@
+        with open(config_file, "w") as f:
+            json.dump(config_copy, f, indent=2)
+
+        # Run training with specified backend using subprocess
+        env = os.environ.copy()
+        cmd = ["dp", "train", config_file]
+        if backend == "pt":
+            cmd = ["dp", "--pt", "train", config_file]
+
+        cmd.extend(["--log-level", "WARNING"])
+
+        result = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)